public/index.html

```diff
@@ -24,7 +24,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <title>PokéLex</title>
```

Comment:
Nice touch!! There are some other meta properties in the head that may be worth updating!